src/components/com_tjdashboard/administrator/sql/install.mysql.utf8.sql

```diff
@@ -46,6 +46,7 @@ CREATE TABLE IF NOT EXISTS `#__tj_dashboard_widgets` (
   `params` text NOT NULL,
   `size` int(11) NOT NULL COMMENT 'Screen size for widget (col-span-3/6/9/12)',
   `autorefresh` int(11) NOT NULL COMMENT 'Widget refresh time span in second',
+  `core` tinyint(1) NOT NULL DEFAULT '0',

```

Comment:
@shitalkakade24 Update the library structure

Reply:
done

Reply:
done